### DISCUSSION TITLE
Will redirection after transfer of ownership persist if the original organisation is deleted?

### DISCUSSION THREAD

**Discussion by ecourtial** [community/community#22029] [Enterprise]
<p>Hi everybody</p>
<p>Context:</p>
<ul>
<li>I work in a company which has a subscription for Github (Github Enterprise).</li>
<li>We forked a public repository some times ago and made it evolve on our own, completely diverging from the original source. The fork is still public.</li>
<li>Today the company will probably close, so we transferred the ownership of our fork to a newly created organization to be able to continue this work after the company closes. The transfer went fine, the redirection works.</li>
</ul>
<p>Question:<br>
when the company will close, and the github subscription cancelled, will the redirection to the new repo after the transfer remain ?</p>

**Comment by queenofcorgis** [comment, marked as answer]
<p>Nice to see you back in the community <a class="mention" href="/ecourtial">@ecourtial</a> :slight_smile:</p>
<p>Yes, the redirection to the new repo after transfer will remain! It will not go away when the organization does.  The only thing that would cause the redirect to be removed is if someone requests it, or if a new repository is created with the same name as the old one (org or username and repo name combination).</p>
